fpx/src/commands/client/requests.rs

```diff
@@ -12,13 +12,21 @@ pub struct Args {
 
 #[derive(Subcommand, Debug)]
 pub enum Command {
-    /// Get a requests
+    /// Get a request
     Get(GetArgs),
+
+    /// List requests
+    List(ListArgs),
+
+    /// Create and execute a request
+    Post(PostArgs),
```

Comment:
Think create would be a better name for this. And giving that we are also executing it, maybe `call` would be better?